experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -308,4 +329,70 @@ export class NodeSDK {
         .then(() => {})
     );
   }
+
+  private configureLoggerProviderFromEnv(): void {
+    const logExportersList = process.env.OTEL_LOGS_EXPORTER ?? '';
+    const enabledExporters = filterBlanksAndNulls(logExportersList.split(','));
+
+    if (enabledExporters.length === 0) {
+      diag.info('OTEL_LOGS_EXPORTER is empty. Using default otlp exporter.');
+      enabledExporters.push('otlp');
+    }
+
+    if (enabledExporters.includes('none')) {
+      diag.info(
+        `OTEL_LOGS_EXPORTER contains "none". Logger provider will not be initialized.`
+      );
+      return;
+    }
+
+    const exporters: LogRecordExporter[] = [];
+
+    enabledExporters.forEach(exporter => {
+      if (exporter === 'otlp') {
+        const protocol = (
+          process.env.OTEL_EXPORTER_OTLP_LOGS_PROTOCOL ??
+          process.env.OTEL_EXPORTER_OTLP_PROTOCOL
+        )?.trim();
+
+        switch (protocol) {
+          case 'grpc':
+            exporters.push(new OTLPGrpcLogExporter());
+            break;
+          case 'http/json':
+            exporters.push(new OTLPHttpLogExporter());
+            break;
+          case 'http/protobuf':
+            exporters.push(new OTLPProtoLogExporter());
+            break;
+          case undefined:
+          case '':
+            diag.info(`OTLP logs protocol is not set. Using http/protobuf.`);
```

Comment:
```suggestion
```

nit: I would drop this. There is no need for `diag.info`-level output for the default, acceptable behaviour. If anything this could be at debug level. However the equivalent env-based handling for the *traces* OTLP protocol does not current log for this case.